src/contracts/atlas/Storage.sol

```diff
@@ -85,6 +85,14 @@ contract Storage is AtlasEvents, AtlasErrors, AtlasConstants {
     /// @return calledBack Boolean indicating whether the solver has called back via `reconcile`.
     /// @return fulfilled Boolean indicating whether the solver's outstanding debt has been repaid via `reconcile`.
     function solverLockData() public view returns (address currentSolver, bool calledBack, bool fulfilled) {
+        return _solverLockData();
```

Comment:
I don't understand the point of this, does it do something? Seems like this wrapping will increase the gas usage by about 88 from my tests.

Reply:
Its more gas efficient to call an internal function than a public one, but we still need to expose that function publically for dapps or solvers to call. But good catch, the outer function should be external not public. Fixed here https://github.com/FastLane-Labs/atlas/pull/225/commits/c1e3e1b9bb56d3ab964f0f0327b6b52662bfb8e7